storage/database/dynamodb.go

```diff
@@ -0,0 +1,479 @@
+package database
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/rcrowley/go-metrics"
+
+	"github.com/pkg/errors"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"github.com/aws/aws-sdk-go/service/dynamodb/dynamodbattribute"
+	"github.com/klaytn/klaytn/log"
+)
+
+var dataNotFoundErr = errors.New("data is not found with the given key")
+var nilDynamoConfigErr = errors.New("attempt to create DynamoDB with nil configuration")
+
+const dynamoWriteSizeLimit = 400*1024 - 32 // The maximum item size is 400 KB including a key size
```

Comment:
32 is the longest key in Klaytn?
How about checking the every sum of key/value for everytime?

Reply:
I think 32 is the longest key since the key is a hash, however I changed the logic to check `len(key) + len(val)` as you commented.